purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/CarouselComponent.kt

```diff
@@ -77,7 +77,17 @@ class CarouselComponent(
         @get:JvmSynthetic
         @SerialName("ms_transition_time")
         val msTransitionTime: Int,
-    )
+        @get:JvmSynthetic
+        @SerialName("transition_type")
+        val transitionType: TransitionType?,
+    ) {
+        @Serializable(with = CarouselTransitionTypeDeserializer::class)
+        enum class TransitionType {
```

Comment:
I started on this before deciding that would be more complex and worth its own PR, but I left it here for now since it was easier and we would need to do this anyway in a followup PR.